app/routes/cookies.js

```diff
@@ -6,13 +6,17 @@ const ViewModel = require('./models/cookies-policy')
 module.exports = [{
   method: 'GET',
   path: '/cookies',
+  options: {
+    auth: false
+  },
   handler: (request, h) => {
     return h.view('cookies/cookie-policy', new ViewModel(request.state[cookiesPolicy], request.query.updated))
   }
 }, {
   method: 'POST',
   path: '/cookies',
   options: {
+    auth: false,
     plugins: {
       crumb: false
     },
```

Comment:
Not related to this PR, but there's no value in a cookie policy in this service.  We probably can strip it all out as part of the ticket we have to make the branding suitable for internal applications.

Reply:
Have gone through and removed the cookie policy code